### ISSUE TITLE
No way to call TimeBudgetPerFrameDeferAgent constructor

### ISSUE THREAD

**Issue by De-Panther** [atteneder/glTFast#119]
As `TimeBudgetPerFrameDeferAgent` is a `MonoBehaviour`, there's no way to call it's constructor and set the `frameBudget`.

**Comment by atteneder** [comment]
Thanks for the report!

I missed it for so long, because I fixed setting up proper notifications only recently. Sorry about that.

You're absolutely right. I fixed the issue in version 2.6.0 and newer by replacing the constructor with a setup method called `SetFrameBudget`.

Now the recommended way to set it (also stated in the [documentation](https://github.com/atteneder/glTFast/blob/main/Documentation~/glTFast.md#tune-loading-performance)) is as follows:

```csharp
var deferAgent = gameObject.AddComponent<TimeBudgetPerFrameDeferAgent>();
// You can then optionally tweak the time budget like so:
// Slower, but less chance of stuttering frame rate:
deferAgent.SetFrameBudget(0.25f);
```

hth

**Comment by De-Panther** [comment]
Thanks!
I ended up creating a custom `IDeferAgent`. So I'll update and replace with the fixed version.

Also, I really like this package. I'm using it in
https://github.com/De-Panther/webxr-input-profiles-loader
where it loads Controllers models in WebXR.
I tried to smooth the loading on as much frames as possible, but it seems that there are still some operations that can't be done over a number of frames.
